proto/orc_proto.proto

```diff
@@ -221,15 +227,32 @@ message PostScript {
   //   [0, 12] = Hive 0.12
   repeated uint32 version = 4 [packed = true];
   optional uint64 metadataLength = 5;
-  // Version of the writer:
-  //   0 (or missing) = original
-  //   1 = HIVE-8732 fixed
-  //   2 = HIVE-4243 fixed
-  //   3 = HIVE-12055 fixed
-  //   4 = HIVE-13083 fixed
-  //   5 = ORC-101 fixed
-  //   6 = ORC-135 fixed
+
+  // The version of the writer that wrote the file. This number is
+  // updated when we make fixes or large changes to the writer so that
+  // readers can detect whether a given bug is present in the data.
+  //
+  // These numbers are assigned from 6 per a writer so that readers that
+  // predate ORC-202 treat the new writers correctly.
```

Comment:
maybe note that writer version is required for everything except for the original version of ORC_JAVA.